_includes/nav.html

```diff
@@ -58,21 +54,18 @@
                </li>
                <li class="open">
                   <span class="notch">&nbsp;</span>
-                  <a href="/community" class="menu-title">Community</a>
+                  <a class="unlinked">Community</a>
                   <ul class="level1">
-                     <li class="leaf"><a href="https://github.com/jbossws/jbossws-cxf/discussions">Discussion</a></li>
-                     <li class="leaf"><a href="https://groups.google.com/forum/#!forum/jbossws">Forums</a></li>
-                     <li class="leaf"><a href="https://www.jboss.org/community/wiki/JBossWS">Wiki(archive)</a></li>
-                     <li class="leaf"><a href="https://community.jboss.org/en/jbossws">User Forum(archive)</a></li>
-                     <li class="leaf"><a href="https://community.jboss.org/en/jbossws/dev">Developer Forum(archive)</a></li>
+                     <li class="leaf"><a href="{{site.baseurl}}/mailinglists">Mailing Lists</a></li>
+                     <li class="leaf"><a href="https://issues.redhat.com/projects/JBWS/issues">Issue Tracker</a></li>
                   </ul>
                </li>
 
                <li class="open">
                   <span class="notch">&nbsp;</span>
-                  <a href="https://issues.redhat.com/browse/JBWS" class="menu-title">Issue Tracker</a>
+                  <a href="https://jira.jboss.org/jira/browse/JBWS" class="menu-title">Issue Tracker</a>
                   <ul class="level1 jiralink">
-                     <li class="leaf"><a href="https://issues.redhat.com/browse/JBWS">JIRA</a></li>
+                     <li class="leaf"><a href="https://jira.jboss.org/jira/browse/JBWS">JIRA</a></li>
```

Comment:
The previous one is the correct JIRA link .

Reply:
ok